chart/values.yaml

```diff
@@ -44,7 +44,7 @@ controller:
     datadog: # datadog cloud provider config
       enabled: true # enable the provider
       ipRangesURL: "https://ip-ranges.datadoghq.com/" # URL to the IP ranges file (format must be the expected one, defaults is the public file provided by the cloud provider)
-  defaultDuration: 1h # default spec.duration for a disruption with none specified
+  defaultDuration: 5m # default spec.duration for a disruption with none specified
```

Comment:
5 minutes seems a bit short for chaos testing we've been performing. Would it be possible to keep this 1h as a default from an end-user default perspective as it's referenced in the docs as such as well?

Perhaps a compromise could be to have something like a local-values.yaml for development specific defaults.

Reply:
I hadn't considered that users were directly using this `values.yaml`, I'll look at adding a separate local dev values